bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java

```diff
@@ -23,8 +24,24 @@
  * through a {@link Bridge}.
  *
  * @author Dennis Nobel - Initial contribution and API
+ * @author Thomas Höfer - Added thing and thing type properties
  */
 public interface Thing {
+    
+    /** the key for the vendor property */
+    public static final String VENDOR = "vendor";
```

Comment:
Could you name all Property Constants "PROPERTY_VENDOR", "PROPERTY_MODEL_ID", etc. please.